oscplot.c

```diff
@@ -5430,44 +5430,37 @@ static inline void marker_set(OscPlot *plot, int i, char *buf, bool force)
 		priv->markers[i].active = TRUE;
 
 	if (priv->markers[i].graph) {
-		strncpy(priv->markers[i].label, buf, 6);
+		snprintf(priv->markers[i].label, sizeof(priv->markers[i].label), buf, i);
```

Comment:
would it be an idea to also const-ify the `char *buf` param ?

Reply:
Yes, but that is a separate change.